integration_test/scene/db_tbl_rw/integration_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/arana-db/arana/test"
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
import style

Reply:
source code was formatted by vs. code, : (

Reply:
> source code was formatted by vs. code, : (

Hi. @csynineyang , you can use `imports-formatter` to format import.

```
go get -u github.com/dubbogo/tools/cmd/imports-formatter

imports-formatter
```

Reply:
got it